app/assets/stylesheets/application.scss

```diff
@@ -12,6 +12,6 @@
  *
  *= require_tree .
  *= require_self
+ *= require_new
```

Comment:
You shouldn't need to add a new line to load the new file - `require_tree .` should load in all files in the stylesheets folder! (While you're at it, move `require foundation_and_overrides` to the top of this little `require` chunk to make sure your styles always take precedence over Foundation's!)